### DISCUSSION TITLE
can i contribute content ?

### DISCUSSION THREAD

**Discussion by AkshayNagamalla** [ML-Fusion-Lab/ML-Fusion-Lab-Website#76] [Q&A] [answered]
Hi @vivekv2810,

I really like the idea of ML-Fusion-Lab! I'm a contributor from GSSoC 2024 extended, and this is my first time participating in open source. I'd love to contribute to the project, especially to the courses section, as I noticed some areas without content. I've been studying machine learning for two years and believe I can help by adding content, explanations, blogs, visualizations, and code to make the concepts easier to understand.

Could you let me know if contributions in this area are welcome? Also, what level would these contributions fall under?

**Comment by vivekvardhan2810** [comment, marked as answer]
Hi @AkshayNagamalla, Your Contributions are welcome.

adding content, explanations, blogs, visualizations, and code to make the concepts easier to understand.

The above ones comes under documentation and we will give the label as level1 contribution.

But you can add this contributions later

I will let you know soon.

if you want to contribute for the website improvements, Your also welcome. (You Can Create issues)

**Reply by AkshayNagamalla** [reply]
Thanks for the opportunity! I'm excited to contribute to the documentation and will wait for your go-ahead to start. In the meantime, I'd be happy to help with website improvements and will create issues as I identify potential enhancements.

Looking forward to working together!
